configuration/README.md

```diff
@@ -37,35 +49,50 @@ FieldTags:
   Val: source
 ```
 
-Sinks and sanitizers are identified via regexp according to package, method, and (optional) receiver name.
+Sinks and sanitizers are identified by package, method, and (optional) receiver name.
+Again, these may be specified by either a provided string literal or regexp.
```

Comment:
For some readers it may not be clear what `Again` is referring to. I think it would be helpful to be more precise, e.g. `Like for Sources, ` or some other formulation referring to the previous section explicitly.